packages/jsxstyle-utils/src/componentStyles.ts

```diff
@@ -6,4 +10,4 @@ export default {
   Row: { display: 'flex', flexDirection: 'row' },
   Col: { display: 'flex', flexDirection: 'column' },
   Grid: { display: 'grid' },
-};
+} as StyleObject;
```

Comment:
Why is this needed? TS should be able to infer a better type for this.

Reply:
@steadicat good call, I’ll remove it.